### PR TITLE
picard CollectWgsMetrics: update to new cli args

### PR DESCRIPTION
**Pull request by matthdsm**
## PR checklist

* Make picard stop complaining about it's changing command line syntax
* drop unused bai input

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by matthdsm** [comment]
Dropped the bai index from the input tuple because it's unused and collides with the other picard metrics modules

File: modules/picard/collectwgsmetrics/main.nf
Comment (sateeshperi) [inline modules/picard/collectwgsmetrics/main.nf:35]: guess will have to update the other picard modules as well in one picard upgrade

**Review by sateeshperi — APPROVED** [review]
lgtm! will wait for another review though

File: modules/picard/collectwgsmetrics/main.nf
Comment (JoseEspinosa) [inline modules/picard/collectwgsmetrics/main.nf:8]: Maybe take the opportunity to bump version `2.27.1`?

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Thanks @matthdsm! though maybe we could bump last Picard version

**Comment by JoseEspinosa** [comment]
> Whoops, missed that one! Should I do a bulk update of all the others too? In a separate PR of course.

If you fancy doing it :smile:

**Review by JoseEspinosa — APPROVED** [review]
Awesome @matthdsm :rocket:

**Pull request merged**
